### DISCUSSION TITLE
How to implement a sound in react?

### DISCUSSION THREAD

**Discussion by PhilipZi** [neuefische/questions#123] [Web] [answered]
Is there a docu which i can use to learn how to implement a sound in form/use of an alert?
i found useSound which i can install as npm install use-sound. Is this a valid source or is there a better way to do this? 
And what are good websites do download a sound from? In my case i need a simple beep. 

Big thanks in advance!

**Comment by JessicaLoers** [comment, marked as answer]
I can highly recommend **[useSound](https://github.com/joshwcomeau/use-sound)**  🙂 which is based on `howler.js` and in my opinion the right  library for your use case. 

There's a also a **[tutorial ](https://www.joshwcomeau.com/react/announcing-use-sound-react-hook/)** with examples and section about how to **[finding and prepping sounds](https://www.joshwcomeau.com/react/announcing-use-sound-react-hook/#finding-and-prepping-sounds)**

**Reply by PhilipZi** [reply]
thanks so i will stick to useSound i just wanted to make sure this is a proper source
